datafusion/expr/src/expr_fn.rs

```diff
@@ -99,6 +99,19 @@ pub fn placeholder(id: impl Into<String>) -> Expr {
     })
 }
 
+/// Create an '*' [`Expr::Wildcard`] expression that matches all columns
+///
+/// # Example
+///
+/// ```rust
+/// # use datafusion_expr::{wildcard};
+/// let p = wildcard();
+/// assert_eq!(p.to_string(), "*")
+/// ```
+pub fn wildcard() -> Expr {
+    Expr::Wildcard { qualifier: None }
+}
```

Comment:
Do we need a `fn wildcard_with_qualifier()` or `fn wildcard(qualifier: None)`? Though it's not used in our examples.

Reply:
I was thinking that it was such a niche usecase that if anyone needed the `qualifier` they could make it explicitly via `Expr::Wildcard{ qualifier: Some(...)}`, but that is indeed messier

I would be happy to add `qualified_wildcard` or similar if others think it would be valuable